### PR TITLE
[api] Miscellaneous consolidation.

### PR DESCRIPTION
**Pull request by ejgallego**
Nothing remarkable, here are some little changes, which make sense on
the large picture of the code organization.

### CODE REVIEW

File: library/library.mllib [hunk 1]
```diff
@@ -1,5 +1,3 @@
-Univops
```
Comment (SkySkimmer) [inline library/library.mllib]: I was wondering about this one. Guess I don't need to now.

Comment (ejgallego) [reply library/library.mllib]: It is moved into engine, not removed. Maybe the natural step is to integrate it into some part of `engine`?

Comment (SkySkimmer) [reply library/library.mllib]: I was precisely wondering why it was in library/.

**Comment by ejgallego** [comment]
I've updated this commit addressing some style comments and bugs made by @maximedenes . It should be ready to go now [travis allowing]

**Comment by maximedenes** [comment]
The linter failed, it seems.

**Comment by ejgallego** [comment]
Yup as this another replace-like API commit. Do you want me to rebase and fix the linter or are you willing to do a dernier exception?

File: library/library.mllib (hunk 1 above)
Comment (mattam82) [reply library/library.mllib]: Due to a dependency problem I couldn’t put it in engine or pretyping (or somewhere in interp I believe), AFAIR

**Comment by maximedenes** [comment]
I have no idea, but you recently said you regretted not having fixed it in previous similar PRs. If you changed your mind a second time, we can make an exception for this one. If not, please fix it.

**Comment by ejgallego** [comment]
I also regret not fixing this one too, but I am a bad busy person so please push if you don't mind.

**Comment by ejgallego** [comment]
Merci monsieur and sorry for all the annoying load.

**Pull request merged**